### ISSUE TITLE
[ENHANCEMENT] WingetUI v3.0.0 UI improvements

### ISSUE THREAD

**Issue by realgooseman** [marticliment/UniGetUI#1958]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
- [X] This improvement is not a bug. If you want to report a bug, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=bug&projects=&template=bug-issue.yml&title=%5BBUG%5D+%28Enter+your+description+here%29)

### Describe the improvement

HI @marticliment,

Thanks for this useful app. I appreciate the work.

I just updated to V3 and I've noticed some improvements, but there are a few things I would like to share about the UI.
I've read some issues where people are complaining about it but without being specific about the things that caught my attention, let me know what you think about it :

### **Text labels**

I don't like that icons get a text label under/next to them. It just takes useless space and could be improved by keeping the buttons icons only and showing the text on hover. This problem is especially annoying on the V3 with the sidebar that shows text labels under all the icons. Also, these icons all have big page titles that clearly indicate what they are.

![brave_fCGW85RMz6](https://github.com/marticliment/WingetUI/assets/64812183/a7f71fa7-b678-462b-baac-bd5062b270ec)

### **Side filter menu**

That new filter menu that you can slide to the left to hide it isn't visually appealing, what about a "filter" button with a pop-up window.

![WingetUI_tpwjooe76E](https://github.com/marticliment/WingetUI/assets/64812183/c2057897-3777-4972-b85a-1a7c9bad4a0d)

### **Remember window position**

On V2, WingetUI used to remember the last window position and size it was left in on launch. 
It doesn't seem to be the case anymore on V3.

### **About button**

The new "about" sidebar button is not really that essential imo, could be moved to the "More" menu to simplify the sidebar

### Describe how this improvement could help users.

In my opinion, it would simplify the interface, make it more visually appealing and consistent.

**Comment by aaronliu0130** [comment]
Personally, having text next to primary actions such as updating makes sense. For the sidebar, maybe it could do the thing where clicking on the menu button shows the text, or shorten everything, like having the text be "Discover, Updates, Installed, Bundles", and only have it show the full thing on hover.

**Comment by tomasz1986** [comment]
Honestly, I like the text labels. Otherwise, I find it always difficult to remember what each icon stands for. The hover state is also not available for those who navigate the application on a touch screen or using only a keyboard. Hiding useful information so that it appears only on hover is bad for accessibility in general.

It could be an opt-in setting though, for those that really want to remove the text labels.

**Comment by aaronliu0130** [comment]
I don't think you can hide it anymore

**Comment by marticliment** [comment]
> I don't think you can hide it anymore

You can drag it to the right on 3.0.0.

However, I will create a show-hide button

**Comment by aaronliu0130** [comment]
> You can drag it to the right on 3.0.0.

How do I do that?

**Comment by marticliment** [comment]
I am sorry, I meant to the left. there is a separator bar used to resize the filter panel, move it to the max left and the panel should hide

**Comment by FrankGarcia97** [comment]
Hi @marticliment, I want to join the discussion, and I have two comments regarding the interface design.

1. This doesn't quite relate to the current topic, but rather to the appearance of the installer itself. However, I didn't want to create a separate topic for this. So, to the point. In my opinion, the existing picture in the installer should be replaced with the application's icon to give it more seriousness. Because currently, it looks somewhat childish for such an app.

![001](https://github.com/marticliment/WingetUI/assets/148349054/509e2139-8138-4fef-b0e7-7bbc6dca4385)


2. Separator on the left looks more like a scroll bar, and that's not very good in my opinion, as it disrupts the coherence. It's worth completely abandoning this, since panels already have their own edges/boundaries, and the size can be adjusted by dragging them, it's more logical. Or at least make this divider not just a short stripe, but a line spanning the entire height of the panel, to make it look more like a separator.

![002](https://github.com/marticliment/WingetUI/assets/148349054/386f207a-4a11-4559-ad68-ff9a8888e1e8)

**Comment by marticliment** [comment]
> 3\. Separator on the left looks more like a scroll bar, and that's not very good in my opinion, as it disrupts the coherence. It's worth completely abandoning this, since panels already have their own edges/boundaries, and the size can be adjusted by dragging them, it's more logical. Or at least make this divider not just a short stripe, but a line spanning the entire height of the panel, to make it look more like a separator.

I'm afraid this is the default WinUI expander, but I'll see what I can do




> 1. This doesn't quite relate to the current topic, but rather to the appearance of the installer itself. However, I didn't want to create a separate topic for this. So, to the point. In my opinion, the existing picture in the installer should be replaced with the application's icon to give it more seriousness. Because currently, it looks somewhat childish for such an app.

New imagery is planned when WingetUI becomes UnigetUI

**Comment by aaronliu0130** [comment]
awwww I liked the wizard, wizard is old, he ain't childish

**Comment by realgooseman** [comment]
Yes, I feel like UnigetUI is redundant. Uniget is a software of itself now with multiple sources (including Winget) + it sounds better.

**Comment by aaronliu0130** [comment]
WingetUI will not use Uniget, and the UI at the end distinguishes it from the other projects that use the Uniget name

**Comment by aaronliu0130** [comment]
I'm not making the decision, I'm saying what I think just like you.
https://github.com/SaladLab/UniGet is in NuGet (and is popular despite being outdated) and then there's https://github.com/uniget-org/cli in Go, though for *nix. Martí has also already stated that they've decided on UnigetUI.

**Comment by marticliment** [comment]
Yes, basically what @aaronliu0130 said is right. And it is still a graphical interface, not a package manager, so UI makes sense.

**Comment by aryanchoudharypro** [comment]
so i want to ask that is the screen reader support being worked in the ui?

**Comment by marticliment** [comment]
As soon as I have the time to
